### ISSUE TITLE
Add "See also"  references

### ISSUE THREAD

**Issue by janosh** [tensorflow/tensorflow#36786]
This is sort of a follow up to #33756. The TF docs have undergone huge improvements over the last couple months. However, one thing I really like about the PyTorch docs which is still (mostly) missing in the TF docs are "**See also**"  references.

A lot of functions have similar or related functionality. Examples include:

1. `tf.split`, `tf.unstack`
2. `tf.size`, `tf.shape`
3. `tf.repeat`, `tf.concat`, `tf.tile`, `tf.stack`
4. `tf.exp`, `tf.math.log`
5. `tf.keras.layers.MaxPool2D`, `tf.nn.max_pool2d `
6. `tf.ones`, `tf.ones_like` and `tf.zeros`, `tf.zeros_like`

just to name a few.

Often people happen to find one and start using it regularly but remain unaware of the others for quite a while. Even if I do know about all of them, I often find myself wanting to compare the signature of similar functions to find the one most suitable to the current use case. In those cases, it takes way too many clicks to get from one to the other(s).

In short, would be great if the docs referenced related content. That should help guide people to use the best tool for the job right from the start.

**Comment by ayushmankumar7** [comment]
I look forward to do this task but i need some guide how to do this. I am a beginner.

**Comment by Rubix982** [comment]
I agree, it would be beneficial to let the user explore the documentations more by finding relevant searches for the user. 
Pull requests have been made as per above. I suggest keeping this issue open so other contributors can suggest other additions as well ( other 'see also' references ) because I'm plenty sure there are more out there.

**Comment by abhinavsp0730** [comment]
Adding on the list.
7. ``` tf.math.multiply ```, ``` tf.math.scalar_mul ```

**Comment by Saduf2019** [comment]
@janosh 
Is this still an issue?

**Comment by janosh** [comment]
I'd say the situation has markedly improved but browsing quickly through the docs, there are some references I listed above that haven't been added yet. Of course it's up to you to decide if you want to have those or not. Missing are

4. `tf.exp`, `tf.math.log`
5. `tf.keras.layers.MaxPool2D`, `tf.nn.max_pool2d `
7. `tf.math.multiply`, `tf.math.scalar_mul` (added by @abhinavsp0730)

which I agree are less important than, say, `tf.repeat`, `tf.concat`, `tf.tile`, `tf.stack`. So feel free to close this if all the references you want are there.

**Comment by MarkDaoust** [comment]
Thanks to everyone who's submitted PRs for this!

We always encourage authors to add these, it's on our internal quality checklists.

If you have anymore PRs please send them to me for review. 

But we don't need to keep this bug open.
